CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 ## 0.1.0
 
 - Initial release.
+
+## 0.2.0
```

Comment:
Mind taking this out of the PR, so that it can be included in the commit for the 0.2.0 tag?